### DISCUSSION TITLE
Clock Generator not found

### DISCUSSION THREAD

**Discussion by SAVE-THE-HERO** [sanni/cartreader#1023] [Need help]
Even if I write “//#define clockgen_installed”, SFC will reply “Clock Generator not found”.
I checked 13.1 and 14.3.　HW3
Is this a problem only for me?
![IMG_6656](https://github.com/user-attachments/assets/3964f28e-271e-4096-b538-893a31a2dc44)
![IMG_6655](https://github.com/user-attachments/assets/eeb9015a-33e5-483e-ad34-68e36f8658bc)

**Comment by sanni** [comment]
You need to disable it here too: https://github.com/sanni/cartreader/blob/9dce89ab05c87126c7d152c2b07f99552fd57ed6/Cart_Reader/Config.h#L570

Edit: Fixed in todays commit.

**Reply by SAVE-THE-HERO** [reply]
I had missed it.
Thank you
